tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectLoadingTest.java

```diff
@@ -79,13 +82,33 @@ public void render () {
 		shapeRenderer.setColor(Color.BLUE);
 		Gdx.gl20.glLineWidth(2);
 		MapLayer layer = map.getLayers().get("Objects");
+		AnimatedTiledMapTile.updateAnimationBaseTime();
 		for (MapObject mapObject : layer.getObjects()) {
-			if (mapObject instanceof TextureMapObject) {
-				TextureMapObject tmObject = (TextureMapObject)mapObject;
+//			if (mapObject instanceof TextureMapObject) {
```

Comment:
We could open an issue to deal with that later but we shouldn't have dead code. Could you please remove it if it's not used?